Dockerfile.doc

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
+
+# We use bash instead of /bin/sh as ESP-IDF doesn't support /bin/sh
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        apt-utils wget software-properties-common gnupg
+
+RUN env DEBIAN_FRONTEND=noninteractive \
+        apt-get update; \
```

Comment:
Is `apt-get update` necessary? It is run in the previous step

Reply:
It is there to work fine with layer caching. It is not an expensive step. If you cache the previous operation and then build a month or two later, `apt-get install` will fail as the APT index would be out-of-date.

Reply:
Then the question becomes why this is not in the original Dockerfile